contracts/issuer-type/src/entry.rs

```diff
@@ -37,14 +37,14 @@ fn parse_issuer_action(args: &Bytes) -> Result<Action, Error> {
 }
 
 fn count_class_cell(args: &Bytes) -> usize {
-    count_cells_with_type_args(Source::Input, &|type_args: &Bytes| {
+    count_cells_with_type_args(Source::Output, &|type_args: &Bytes| {
         type_args.len() == CLASS_TYPE_ARGS_LEN && type_args[0..ISSUER_TYPE_ARGS_LEN] == args[..]
     })
 }
 
 fn handle_creation(args: &Bytes) -> Result<(), Error> {
-    let out_point = load_input_out_point(0, Source::Input)?;
-    if args[..] != blake2b_160(out_point.as_slice()) {
+    let first_input = load_input(0, Source::Input)?;
+    if args[..] != blake2b_160(first_input.as_slice()) {
```

Comment:
should the same generation method as TypeID args be used here, right?

https://github.com/nervosnetwork/ckb/blob/79d619d6643d5dadb3376743afa799b4dfe84ddd/script/src/type_id.rs#L68-L69

Reply:
I will update the code later.